src/include/daos_srv/dtx_srv.h

```diff
@@ -79,7 +79,9 @@ struct dtx_handle {
 					 /* dti_cos has been committed. */
 					 dth_dti_cos_done:1,
 					 /* XXX: touch ilog entry. */
-					 dth_has_ilog:1;
+					 dth_has_ilog:1,
+					 /* epoch conflict, need to renew. */
```

Comment:
It's to restart dtx transaction? so dth_dtx_restart looks better to me.

Reply:
Only restart DTX is NOT enough, we need new epoch, that is why call it as "renew".